infra/tf/main.tf

```diff
@@ -71,42 +70,37 @@ resource "digitalocean_droplet" "bootstrap-node" {
   size       = var.droplet_size
   backups    = false
   monitoring = true
-  user_data  = file(format("%s/bootstrap_node.yml", path.module))
+  user_data  = file("${path.module}/configs/bootstrap_node.conf")
   ssh_keys = [
     data.digitalocean_ssh_key.ursa-dev.id
   ]
 }
 
 resource "digitalocean_firewall" "ursa-network" {
-  count = var.node_count
+  count = 1
```

Comment:
```suggestion
  count = var.node_count
```

Reply:
So the reason I set this to 1 is:
- the existing DO deployment only had a single firewall
- a few lines down, for droplet_ids, it selects all bootstrap and testnet nodes. If we want an individual firewall for each, it should be configured as such, otherwise we end up with 7 firewalls on every node.